src/main/java/pl/gispartner/ResourceManagerApp/api/ResourceController.java

```diff
@@ -0,0 +1,25 @@
+package pl.gispartner.ResourceManagerApp.api;
+
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import pl.gispartner.ResourceManagerApp.model.ResourceEntity;
+import pl.gispartner.ResourceManagerApp.service.ResourceService;
+
+@RestController
+@RequestMapping("/api/resource")
+@RequiredArgsConstructor
+public class ResourceController {
+
+    private final ResourceService resourceService;
+
+    @PostMapping("/save")
+    public void saveResource (@RequestBody ResourceEntity resourceEntity){
```

Comment:
przerob takie miejsca jak to zeby wartstwa perzystencji (encja) nie przenikala do frontu, unika sie takiego podejscia, podaj tu np. ResourceRequest ktory bedzie zawieral takie same pola jak encja a nastepnie przemapuj builderem (builder pattern) na obiekt bazodanowy (encja)